apps/docs-snippets/src/guide/provider/provider.test.ts

```diff
@@ -19,6 +19,8 @@ function decorateResponseWithCustomLogic(response: Response) {
  * @group node
  * @group browser
  */
+
+// #TODO: Do we want these to use launchTestNode ?
```

Comment:
Given that these tests are to evaluate the behaviour of an actual fuel-core node, do we want to replace them with `launchTestNode` ?

Reply:
These snippets are showcasing `Provider` options, there's nothing related to _settting up_ a fuel-core node so it's okay to use `launchTestNode`. We'd add the `launchTestNode` before the `#region`s and set  `FUEL_NETWORK_URL` to the node's url.

Reply:
That's true, I was trying to think of a scenario where we our tests would not catch a disparity between the `Provider` class and `launchTestNode` but I can't think of any.

4abbb87b8